iidm/iidm-serde/src/main/resources/xsd/iidm_V1_13.xsd

```diff
@@ -703,18 +702,13 @@
             </xs:extension>
         </xs:complexContent>
     </xs:complexType>
-    <xs:complexType name="AreaType">
-        <xs:complexContent>
-            <xs:extension base="iidm:Identifiable"/>
-        </xs:complexContent>
-    </xs:complexType>
     <xs:complexType name="VoltageLevelRef">
         <xs:attribute name="id" use="required" type="iidm:nonEmptyString"/>
     </xs:complexType>
-    <xs:complexType name="BoundaryTerminal">
-        <xs:sequence>
-            <xs:element name="terminalRef" type="iidm:TerminalRef"/>
-        </xs:sequence>
+    <xs:complexType name="AreaBoundary">
+        <xs:attribute name="type" use="required" type="iidm:nonEmptyString"/>
+        <xs:attribute name="id" use="required" type="iidm:nonEmptyString"/>
+        <xs:attribute name="side" type="iidm:Side"/>
```

Comment:
It is needed in case of a Terminal used as a boundary. We identify the terminal by the id of the connectable and a side if it is a branch. 
We use the methods of the terminalRef serde but wanted to avoid nesting for just one TerminalRef.

Reply:
OK, you are right. I missed this.
Thanks.